### PR TITLE
Fixed the linebreak problem on debug mode.

### PR DESCRIPTION
**Pull request by jefersonralmeida**
* Replaced the `PHP_EOL . PHP_EOL` with `\n\n`.
* Included a little logic to normalize the line breaks to be always `\n`.

### CODE REVIEW

**Review by tomfor — DISMISSED** [review]
LGTM

**Review by czarpino — DISMISSED** [review]
LGTM

File: fccloudapi.php [hunk 1]
```diff
@@ -2566,7 +2569,8 @@ protected function afterRequestDebug(string $method, string $url, string $postDa
 
             // request headers
             $rawRequest = curl_getinfo($this->curl_handle, CURLINFO_HEADER_OUT);
-            $lines = explode(PHP_EOL, trim($rawRequest));
+            $rawRequest = str_replace('\r\n', '\n', trim($rawRequest));
+            $lines = explode('\n', trim($rawRequest));
```
Comment (czarpino) [inline fccloudapi.php]: Aren't we trimming a bit too much? 😅

Comment (jefersonralmeida) [reply fccloudapi.php]: Yes!! I'll fix it..., lol

**Review by tomfor — APPROVED** [review]
LGTM

**Review by czarpino — APPROVED** [review]
Looks better! 😆

**Pull request merged**